src/segment.js

```diff
@@ -71,14 +72,17 @@ define([
    */
 
   function Segment(peaks, id, startTime, endTime, labelText, color, textColor, opacity, editable) {
+    var minSize = 0.5;
```

Comment:
It must be 0.35. I was wrong when I told you 0.5 initially. We picked 0.35 with Florian.

Reply:
Also, make a constant instead of a variable.
```
var MIN_SEGMENT_SIZE = 0.35
```

Reply:
It's done